R/position.clones.R

```diff
@@ -81,53 +81,62 @@ position.clones <- function(v, tree, wid) {
     }
 
 position.nodes.fixed <- function(v, tree, fixed.angle, len) {
+    calculate.angles <- function(v, tree, fixed.angle) {
+        node.ids <- c(v$id[[1]]);
+        
+        while (length(node.ids) > 0) {
+            current.node.id <- node.ids[1];
+            node.ids <- node.ids[-1];
+
+            child.ids <- tree$tip[tree$parent == current.node.id & !is.na(tree$parent)];
+
+            if (length(child.ids) > 0) {
+                # Safe to hardcode temporarily. as this will only ever apply to 
```

Comment:
As this will only ever apply to... what?

Reply:
It's a mystery! Fixed [here](https://github.com/uclahs-cds/package-CancerEvolutionVisualization/blob/d33706a598bb2a6fff97a5d381f2cb9145178b25/R/position.clones.R#L94-L99_).